.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: java
 jdk:
   - oraclejdk8
+  - oraclejdk9
```

Comment:
this will double the required number of executors on Travis:(

I would rather add 1 stage like `docker-in-alpine-docker` but with `openjdk:9-jdk-alpine`

Reply:
Afaik this does not double the required number of executors, since it seems it's not applied as a matrix configuration in this case, but will only duplicate the 
```
install:
  - echo "MAVEN_OPTS='-XX:+TieredCompilation -XX:TieredStopAtLevel=1'" > ~/.mavenrc
  - ./mvnw -T4 -DskipTests=true -Dmaven.javadoc.skip=true install
```
step and not the individual jobs. At least it looks like this in Travis.

Reply:
I also wonder why this step runs tests on case of jdk9?

Reply:
Okay, travis seems to have added a new default job for both JDKs instead of applying it to the build matrix, I'll try something else instead.